week1/reposts/maverickchaser/Main.hs

```diff
@@ -0,0 +1,34 @@
+import Control.Monad
+import System.Random
+
+numberOfReposters       = 1000000
+numberOfFakeAccounts    = 100000
+numberOfSimulations     = 1000
+numberOfWinners         = 7
+
+
+generateAccounts :: Int -> Int -> StdGen -> [Int]
+generateAccounts nAccounts maxAccId gen =
+    take nAccounts $ randomRs (1, maxAccId) gen :: [Int]
```

Comment:
You don't need `:: [Int]` here, it will be inferred from the top-level type signature for `generateAccounts`.